glutin/src/config.rs

```diff
@@ -166,7 +166,14 @@ impl ConfigTemplateBuilder {
 
     /// The set of apis that are supported by configuration.
     ///
-    /// By default api isn't specified when requesting the configuration.
+    /// The default is specific based on the used API. If you
```

Comment:
Not quite sure how to word this because we have the `Api` flag here which is different from the EGL/GLX/WGL/CGL "API", but they are named the same... Wikipedia calls EGL a "native graphics platform interface".

```suggestion
    /// The default [`Api`] depends on the used graphics platform interface. If you
```

But not really liking that yet.

Reply:
I think it's fine, at least it's better than what we had.